Formula/prs.rb

```diff
@@ -0,0 +1,32 @@
+class Prs < Formula
+  desc "Secure, fast & convenient password manager CLI with GPG & git sync"
+  homepage "https://timvisee.com/projects/prs"
+  url "https://github.com/timvisee/prs/archive/v0.4.1.tar.gz"
+  sha256 "f7f8f5d815cf1c4034f1c2aa36ed29b7e3ab2a884791b4b69ffc845d5d111524"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "gpgme"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@3"
+  end
+
+  def install
+    cd "cli" do
+      system "cargo", "install", *std_cargo_args
+    end
```

Comment:
Does this work?
```suggestion
    system "cargo", "install", *std_cargo_args(root: "./cli")
```

Reply:
No, but `*std_cargo_args(path: "./cli")` does work!